debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/RecordsSnapshotProducerIT.java

```diff
@@ -424,4 +424,40 @@ public void shouldGenerateSnapshotsForPartitionedTables() throws Exception {
             assertSourceInfo(record);
         }
     }
+    @Test
+    // HStores process as HashMaps internally for Snapshot producer only (not stream...)
```

Comment:
Please instead of comment use `FixFor` annotation

Reply:
Done 👍